server/odc-service/src/main/java/com/oceanbase/odc/config/ScheduleConfiguration.java

```diff
@@ -168,7 +168,7 @@ public ThreadPoolTaskExecutor cloudLoadDataTaskExecutor() {
     @Bean(name = "syncDatabaseTaskExecutor")
     public ThreadPoolTaskExecutor syncDatabaseTaskExecutor() {
         ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
-        int poolSize = Math.max(SystemUtils.availableProcessors(), 5);
+        int poolSize = 64;
```

Comment:
avoid hardcode, extract to system configuration, 
or  something like `max (SystemUtils.availableProcessors() * 8,  64)`

Reply:
fixed